tldr/daemon/startup.py

```
@@ -264,21 +266,61 @@ def start_daemon(project_path: str | Path, foreground: bool = False):
+                            stdout=subprocess.DEVNULL,
+                        )
+                        print(f"Daemon started with PID {proc.pid}")
+                        print(f"Listening on {addr}:{port}")
+
+                    finally:
+                        # Release lock
+                        try:
+                            msvcrt.locking(lock_file.fileno(), msvcrt.LK_UNLCK, 1)
+                        except OSError:
+                            pass
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)